Modules/Core/Common/include/itkBuildInformation.h

```diff
@@ -33,7 +33,7 @@ namespace itk
  * that describe this version of ITK.
  *
  * Values in in the map provide information about
- * the ITK (i.e. it's version, the build options,
```

Comment:
@jhlegarreta Thanks, however, the term "it's" can be found 171 times in ITK cxx, h, and hxx files. Including many similar typo's. Do you intend to fix them as well?

Personally I would rather not have an entire pull request dedicated to just a single occurrence of such a very tiny typo.

Reply:
@N-Dekker No, I do not intend to inspect them all and see if they need to be fixed. I was looking at this file and sport the typo. As always, a PR accepts improvement commits, and amending commits if applicable, from anybody.

Reply:
@jhlegarreta Is this the only typo you spot in this source file?

Reply:
I created #2997 to track this.